Polygon_mesh_processing/test/Polygon_mesh_processing/measures_test.cpp

```diff
@@ -218,7 +359,11 @@ int main(int argc, char* argv[])
   // It won't work with Epec for large meshes as it builds up a deep DAG
   // leading to a stackoverflow when the destructor is called.
   test_centroid<CGAL::Surface_mesh<Epic::Point_3>,Epic>(filename_surface_mesh);
-
+  test_compare<CGAL::Polyhedron_3<Epic> >();
```

Comment:
Needs a test for two different PolygonMesh types since you chose two different template parameters/